src/webviews/webview-side/interactive-common/mainState.ts

```diff
@@ -57,8 +57,6 @@ const darkStyle = `
             --code-string-color: #ce9178;
             --code-variable-color: #9CDCFE;
             --code-type-color: #4EC9B0;
-            --code-font-family: Consolas, 'Courier New', monospace;
```

Comment:
why are so many font stylings being removed?

Reply:
Because they're not supported anymore since we got rid of the CSS generator. This was a fallback but the default fallback of '--vscode-font-family' is more appropriate.